lua/autorun/server/sv_pvp_movespeed.lua

```diff
@@ -46,20 +47,21 @@ local function playerIsInBuild( ply )
     return !getPlayerPvpMode( ply )
 end
 
-local function movementMultiplier( weaponCount )
-    if weaponCount < 1 then return 1 end
-    return math.Clamp(1 - (1.9^( weaponCount))/100, 0, 1)
+local function movementMultiplier( totalWeight )
+    if totalWeight < 1 then return 1 end
+    return math.Clamp(1 - (1.9^( totalWeight))/100, 0, 1)
 end
 
-local function setSpeedFromWeaponCount( ply, weaponCount )
-    local multiplier = movementMultiplier( weaponCount )
+local function setSpeedFromWeight( ply, totalWeight )
+    local multiplier = movementMultiplier( totalWeight )
 
     local newRunSpeed = baseRunSpeed * multiplier
     local newWalkSpeed = baseWalkSpeed * multiplier
     ply:SetRunSpeed( math.max( newRunSpeed, minRunSpeed ) )
     ply:SetWalkSpeed( math.max( newWalkSpeed, minWalkSpeed ) )
 
     if newWalkSpeed < 100 then
+        ply:ChatPrint("You are holding too many weapons drop some to regain speed")
```

Comment:
"You are holding too many weapons! Drop some to regain speed."

Reply:
"You are holding too many weapons, drop some to regain speed!"

Reply:
I like it with a period